data/dataset-curation.md

```diff
@@ -50,9 +50,23 @@ sub-001
 
 Many kinds of data have a place specified for them by BIDS. See [file naming conventions](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure) and the [MRI](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) and [Microscopy](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/10-microscopy.html) extensions for full details.
 
-```{warning}
-TODO: describe neuropoly-specific BIDS entities, like bp-cspine or acq-MTon
+ℹ️ If you need to differentiate spinal cord images from the brain, use the `acq-cspine` tag. For example, `sub-001_acq-cspine_T1w.nii.gz`.
```

Comment:
use [admonitions](https://github.com/neuropoly/intranet.neuro.polymtl.ca/wiki#admonitions) for notes

Reply:
Thank you! Switched in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/94/commits/6e92763bc28ea009ce2cb7f7e65df0911de8f2a9